app/src/main/java/com/amaze/filemanager/adapters/RecyclerAdapter.java

```diff
@@ -834,6 +834,7 @@ private void showThumbnailWithBackground(ItemViewHolder viewHolder, IconDataParc
             @Override
             public boolean onLoadFailed(@Nullable GlideException e, Object model, Target target, boolean isFirstResource) {
                 new Handler(msg -> {
+                    viewHolder.genericIcon.setVisibility(View.VISIBLE);
```

Comment:
This seems redundant, it is done in line 828.

Reply:
Nope, doesn't work without this. Can try.